vg/vgimg/vgimg.go

```diff
@@ -161,7 +166,16 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 		c.img = img
 		c.gc = gc
 		c.dpi = gc.GetDPI()
-		return setsDPI | setsSize
+		return setsDPI | setsSize | setsBackground
+	}
+}
+
+// UseBackgroundColor specifies the image background color.
+// Without UseBackgroundColor, the default color is white.
```

Comment:
I've just realised that this is not necessarily true. If a user calls `NewWith()` without `UseBackgroundColor`, then the background color is nil. For this to be true, `c.backgroundColor` should be set to ` image.White` before the options loop.

After that, LGTM.

Reply:
done